packages/tasks/src/tasks/image-classification/about.md

```diff
@@ -16,12 +16,14 @@ With the `transformers` library, you can use the `image-classification` pipeline
 
 ```python
 from transformers import pipeline
-clf = pipeline("image-classification")
-clf("path_to_a_cat_image")
-
-[{'label': 'tabby cat', 'score': 0.731},
-...
-]
+image_classifier = pipeline(task="image-classification", model="microsoft/resnet-50")
+image_classifier("path_to_a_cat_image")
+
+[{'label': 'Egyptian cat', 'score': 0.7005051970481873},
+ {'label': 'tabby, tabby cat', 'score': 0.16163259744644165},
```

Comment:
again I'd remove this one and below two lines and only keep first and last for better readability and less noise